docs/development/contributing-to-docs.md

```diff
@@ -0,0 +1,139 @@
+  ~~~~
+  [remote "upstream"]
+  	url = https://github.com/apache/druid.git
+  	fetch = +refs/heads/*:refs/remotes/upstream/*
+  	pushurl = no_push
+  [branch "master"]
+  	remote = upstream
+  	merge = refs/heads/master
+  [remote "origin"]
+  		url = https://github.com/{my-git-id}/druid.git
+  		fetch = +refs/heads/*:refs/remotes/upstream/*
+  [branch "master"]
+  		remote = origin
+  		merge = refs/heads/master
+  ~~~~
+
+  For ```upstream```, ```push_url = no_push``` means you won’t accidentally push to upstream.
+  Make sure to put your github id for {my-git-id}.
+  4. ```git config --list --show-origin``` to make sure you’ve got your email configured. If you need to set your email, you can set it per repo or globally. Global instructions [here](https://docs.github.com/en/github-ae@latest/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git).
+  5. Docusaurus?
+
+## Contributing
+
+  1. On branch ```master```, fetch the latest commit:
+
+  ~~~~
+  git fetch upstream
+
+  remote: Enumerating objects: 397, done.
+  remote: Counting objects: 100% (341/341), done.
+  remote: Compressing objects: 100% (181/181), done.
+  remote: Total 397 (delta 118), reused 255 (delta 98), pack-reused 56
+  Receiving objects: 100% (397/397), 266.19 KiB | 16.64 MiB/s, done.
+  Resolving deltas: 100% (118/118), completed with 44 local objects.
+  From https://github.com/apache/druid
+     819d706082..6c9f926e3e  0.21.0       -> upstream/0.21.0
+     8a3be6bccc..84aac4832d  master       -> upstream/master
+   * [new tag]               druid-0.21.0 -> druid-0.21.0
+
+  ➜ git reset --hard upstream/master
+  HEAD is now at 84aac4832d Add feature to automatically remove rules based on retention period (#11164)
+  ~~~~
+
+  Now you're up to date.
+
+  2. Create your working branch:
+  ~~~~
+  git checkout -b my-work
+  ~~~~
+  3. Make your changes, add, and commit:
+  ~~~~
+  git add my-change.md
+  git commit -m "i made some changes"
+  ~~~~
+  4. Test changes locally.
+  5. Push your changes to your fork: ```origin```
+  ~~~~
+  git push --set-upstream origin my-work
+  ~~~~
+  6. Go to GitHub for the Druid repo. It should realize you have a new branch in your fork. Create a pull request from your for ```my-work``` to ```master``` in the Druid repo.
+
+## Style guide
+
+Before publishing new content or updating an existing topic, audit your documentation using this checklist to make sure your contributions align with existing documentation.
+
+* Use descriptive link text. If a link downloads a file, make sure to indicate this action
+* Use present tense where possible
+* Avoid negative constructions when possible
+* Use clear and direct language
+* Use descriptive headings and titles
+* Avoid using a present participle or gerund as the first word in a heading or title
```

Comment:
Example for those of us who've forgotten this stuff from English class?

Reply:
```suggestion
* Avoid using a present participle or gerund as the first word in a heading or title. A shortcut for this is to not start with a word that ends in `-ing`. For example, don't use "Configuring Druid." Use "Configure Druid."
```